odoo/addons/base/migrations/11.0.1.3/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,429 @@
+base         / ir.actions.report        / groups_id (many2many)         : NEW relation: res.groups
+base         / ir.actions.report        / help (html)                   : NEW 
+base         / ir.actions.report        / model (char)                  : NEW required: required
+base         / ir.actions.report        / multi (boolean)               : NEW 
+base         / ir.actions.report        / name (char)                   : NEW required: required
+base         / ir.actions.report        / paperformat_id (many2one)     : NEW relation: report.paperformat
+base         / ir.actions.report        / print_report_name (char)      : NEW 
+base         / ir.actions.report        / report_file (char)            : NEW 
+base         / ir.actions.report        / report_name (char)            : NEW required: required
+base         / ir.actions.report        / report_type (selection)       : NEW required: required, selection_keys: ['qweb-html', 'qweb-pdf'], req_default: function
+base         / ir.actions.report        / type (char)                   : NEW required: required, req_default: function
+# DONE: pre-migration: rename columns
+# - ir_values_id
+# - header
+# - parser
+# - auto
+# - report_xsl
+# - report_xml
+# - report_rml
+# - report_sxw_content_data
+# - report_rml_content_data
+# DONE: pre-migration: copy column report_type
+# DONE: pre-migration: rename model
+# TODO: post-migration: map values 'controller', 'pdf', 'sxw' and 'webkit' of report_type to default value
+
+base         / ir.actions.server        / action_id (many2one)          : DEL relation: ir.actions.actions
+# DONE: pre-migration: rename column
+
+base         / ir.actions.server        / binding_model_id (many2one)   : NEW relation: ir.model
+base         / ir.actions.server        / binding_type (selection)      : NEW required: required, selection_keys: ['action', 'report'], req_default: function
+# NOTHING TO DO
+
+base         / ir.actions.server        / condition (char)              : DEL 
+base         / ir.actions.server        / copyvalue (char)              : DEL 
+base         / ir.actions.server        / id_object (reference)         : DEL 
+base         / ir.actions.server        / id_value (char)               : DEL 
+base         / ir.actions.server        / link_new_record (boolean)     : DEL 
+base         / ir.actions.server        / menu_ir_values_id (many2one)  : DEL relation: ir.values
+base         / ir.actions.server        / model_object_field (many2one) : DEL relation: ir.model.fields
+base         / ir.actions.server        / ref_object (reference)        : DEL 
+# DONE: pre-migration: rename column
+
+base         / ir.actions.server        / state (selection)             : selection_keys is now '['code', 'multi', 'object_create', 'object_write']' ('function')
+# DONE: pre-migration: copy column
+# TODO: post-migration: map values 'trigger' and 'client_action' to default value
```

Comment:
Leave `trigger` value, so that it can be identified as invalid (there are no workflows anymore in Odoo 11). Leave client_action entries as well. They should be removed by the migration of ir.actions.server, and other entries should be recognizable as obsolete.

Reply:
Done.